runtime/make/Makefile.runtime.jemalloc-jemalloc

```diff
@@ -0,0 +1,21 @@
+# Copyright 2004-2017 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RUNTIME_INCLS += -I$(JEMALLOC_INCLUDE_DIR)
+# Chapel third-party install of jemalloc calls the routines
+# chpl_je_malloc, chpl_je_free, etc
+RUNTIME_DEFS += -DCHPL_JEMALLOC_PREFIX=chpl_je_
```

Comment:
Move this into third-party/jemalloc/Makefile.include and use it to set `--with-jemalloc-prefix=`?